raven/src/intTest/java/org/opencadc/raven/FilesTest.java

```diff
@@ -110,6 +122,8 @@ public class FilesTest extends RavenTest {
     private static final URI RESOURCE_ID1 = URI.create("ivo://negotiation-test-site1");
     private static final URI RESOURCE_ID2 = URI.create("ivo://negotiation-test-site2");
 
+    private static final URI CONSIST_RESOURCE_ID = URI.create("ivo://cadc.nrc.ca/minoc");
```

Comment:
This is the default local minoc test instance: that's fine. However, it is critical that this service and the raven service being tested use a different backend database. Otherwise, when the PUT in the consistency test finished, the artifact will be in raven's database (presumably without a siteLocation)... not sure what would happen but it would be easy for incorrect test system setup to be really confusing. I sometimes re-use the back end database for different things and while I do have a separate minoc db it was for something else and this is a subtle new requirement in raven tests.

Please update src/intTest/README.md to make it clear that `ivo://cadc.nrc.ca/minoc` and `ivo://cadc.nrc.ca/raven` *must* have distinct back end database.